sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Internals/LiveMetrics/Diagnostics/LiveMetricsExporterEventSource.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+using Azure.Monitor.OpenTelemetry.AspNetCore.Models;
+
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Internals.LiveMetrics.Diagnostics
+{
+    /// <summary>
+    /// EventSource for the AzureMonitorExporter.
+    /// EventSource Guid at Runtime: 72f5588f-fa1c-502e-0627-e13dd2bd67c9.
+    /// (This guid can be found by debugging this class and inspecting the "Log" singleton and reading the "Guid" property).
+    /// </summary>
+    /// <remarks>
+    /// PerfView Instructions:
+    /// <list type="bullet">
+    /// <item>To collect all events: <code>PerfView.exe collect -MaxCollectSec:300 -NoGui /onlyProviders=*OpenTelemetry-AzureMonitor-LiveMetrics</code></item>
+    /// <item>To collect events based on LogLevel: <code>PerfView.exe collect -MaxCollectSec:300 -NoGui /onlyProviders:OpenTelemetry-AzureMonitor-LiveMetrics::Verbose</code></item>
+    /// </list>
+    /// Dotnet-Trace Instructions:
+    /// <list type="bullet">
+    /// <item>To collect all events: <code>dotnet-trace collect --process-id PID --providers OpenTelemetry-AzureMonitor-LiveMetrics</code></item>
+    /// <item>To collect events based on LogLevel: <code>dotnet-trace collect --process-id PID --providers OpenTelemetry-AzureMonitor-LiveMetrics::Verbose</code></item>
+    /// </list>
+    /// Logman Instructions:
+    /// <list type="number">
+    /// <item>Create a text file containing providers: <code>echo "{72f5588f-fa1c-502e-0627-e13dd2bd67c9}" > providers.txt</code></item>
+    /// <item>Start collecting: <code>logman -start exporter -pf providers.txt -ets -bs 1024 -nb 100 256</code></item>
+    /// <item>Stop collecting: <code>logman -stop exporter -ets</code></item>
+    /// </list>
+    /// </remarks>
+    [EventSource(Name = EventSourceName)]
+    internal sealed class LiveMetricsExporterEventSource : EventSource
```

Comment:
Do we need multiple eventsource in distro? 
I would like to get this merged, we could consider in a follow up PR.

Reply:
>Do we need multiple eventsource in distro?

It's debatable. You said we _might_ move this to Exporter at some point in the future. 
If we want to prepare for that, best to keep it separate.

Either way, I recommend merging in a follow up PR to keep this PR as simple as possible.

Reply:
Check if this can be a partial class of existing event source, instead of moving all the code.

> You said we might move this to Exporter at some point in the future.

This is uncertain, we cannot change all of design needs for it. A minimal change might be needed if we have to move.

Reply:
Spoke offline. Above all other concerns, we don't want to have two EventSource guids in this library as that will create supportability headaches. We will merge these files in a follow up PR.